app/src/main/java/ch/epfl/sdp/replay/LocalReplayStore.kt

```diff
@@ -10,20 +10,20 @@ import javax.inject.Inject
 
 class LocalReplayStore(private val context: Context) {
     @Inject lateinit var myAppDatabase: AppDatabaseCompanion
-    private val REPLAY_DIR = File(context.cacheDir.absolutePath + "/replays")
+    private val REPLAYDIR = File(context.cacheDir.absolutePath + "/replays")
```

Comment:
Why this refactoring? it is less readable like that

Reply:
C'est un Lint Kotlin. Pas d'underscore dans les noms en capslock